dali/test/python/test_external_source_parallel_custom_serialization.py

```diff
@@ -165,35 +161,38 @@ def py_file_gen_reader():
     return py_file_gen_reader
 
 
-jpeg_file = os.path.join(get_dali_extra_path(), 'db', 'single', 'jpeg', '510', 'ship-1083562_640.jpg')
+jpeg_file = os.path.join(get_dali_extra_path(), 'db', 'single', 'jpeg', '510',
+                         'ship-1083562_640.jpg')
```

Comment:
```suggestion
jpeg_file = os.path.join(
    get_dali_extra_path(), 'db', 'single', 'jpeg', '510', 'ship-1083562_640.jpg')
```
Path elements in one line look better.

Reply:
Done